static/js/brand-store/SectionNav/SectionNav.test.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import SectionNav from "./SectionNav";
+
+jest.mock("react-router-dom", () => ({
```

Comment:
Instead of mocking this we've instead used `MemoryRouter` from `react-router` when we don't need to interact with the route and used `createMemoryHistory` from the `history` package passed to the `Router` when we do. It allows us to be a bit more explicit in our tests.